### ISSUE TITLE
Attempting to use bind_to = "private_ip" crashes converge

### ISSUE THREAD

**Issue by tbunnyman** [phlipper/chef-percona#204]
CentOS 6.5

```
       NoMethodError
       -------------
       undefined method `find_private_ip' for Percona::ConfigHelper:Module
```

Here is the dump of the full error
https://skwrl.es/g89q45f63kvmjp0.txt

**Comment by vermut** [comment]
Same on ubuntu 14.04
Chef: 11.12.8

**Comment by phlipper** [comment]
This is resolved via #264.
